CDCgov/cdc-open-viz
# feature: [DEV-7047] Multi Visualization Component

Storybook path: Pages/Dashboard/Multi Visualization


Testing steps from Editor:

Scenario: Tester has selected a dataset and have selected dashboard as the visualization type. They are now on the dashboard screen ready to apply the dataset to widgets.

1) drag and drop a table component on to the default widget row.
2) Configure the data for the row:
    option a) select the gear icon on the widget itself and select "apply to row" then continue the data configuration as normal.
    option b) select the gear icon on the row itself and configure the data.
3) before finishing the data configuration check the box that says: "Configure Multiple Visualizations"
4) when the dropdown appears select a category column you'd like to use to repeat your visualization.
5) complete the configuration.
6) navigate to the dashboard preview page.

Expected: You should see a table for each item in the column category that you selected.
